stdlib/CRC32c/test/runtests.jl

```diff
@@ -64,6 +90,19 @@ function crc32c_sw(s::Union{String, SubString{String}}, crc::UInt32=0x00000000)
     unsafe_crc32c_sw(s, sizeof(s), crc)
 end
 
+function crc32c_sw(a::AbstractVector{UInt8}, crc::UInt32=0x00000000)
+    last = lastindex(a)
+    nb = length(a)
+    buf = Vector{UInt8}(undef, min(nb, 24576))
+    while nb > 0
+        n = min(nb, 24576)
+        copyto!(buf, 1, a, last - nb + 1, n)
+        crc = unsafe_crc32c_sw(buf, n % Csize_t, crc)
+        nb -= n
+    end
+    return crc
+end
+
```

Comment:
Maybe just:
```suggestion
crc32c_sw(a::AbstractVector{UInt8}, crc::UInt32=0x00000000) =
    crc32c_sw(copyto!(Vector{UInt8}(undef, length(a)), a))
```
Memory-efficiency shouldn't matter here since this is just a test method?